src/ui.rs

```diff
@@ -181,5 +182,31 @@ impl ViewState {
         let text = Text::from(text);
         let paragraph = Paragraph::new(text).block(right_block_down);
         f.render_widget(paragraph, right_chunks[1]);
+
+        if show_help {
+            let block = Block::default().title("Help").borders(Borders::ALL);
```

Comment:
Block::new is const, default isnt. bordered is shorter and const. Can be applied in other places too :upside_down_face: 

```suggestion
            let block = Block::bordered().title("Help");
```

Reply:
Thanks! I fixed it!